Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -45,8 +45,11 @@
 "action-saved-accessibility" = "Saved. Activate to unsave.";
 "action-share" = "Share";
 "action-unsave" = "Unsave";
+"adjust-text-sizing" = "Adjust text sizing";
```

Comment:
May want to consider renaming "adjust-text-sizing" and "adjust-text-sizing-footer" to start with "appearance-settings-".